sql/sql_yacc.yy

```diff
@@ -16380,40 +16379,6 @@ alter_instance_action:
             if (is_identifier($2, "REDO"))
             {
               $$= NEW_PTN PT_alter_instance(ROTATE_REDO_SYSTEM_KEY, EMPTY_CSTR, 0);
-||||||| merged common ancestors
-              $$= NEW_PTN PT_alter_instance(ROTATE_BINLOG_MASTER_KEY, EMPTY_CSTR, 0);
-            }
-            else
-            {
-              YYTHD->syntax_error_at(@2);
-              MYSQL_YYABORT;
-            }
-          }
-          | ROTATE_SYM ident_or_text SYSTEM_SYM KEY_SYM ulong_num
-          {
-            if (is_identifier($2, "INNODB"))
-            {
-              if ($5 > UINT_MAX32 - 1)
-              {
-                my_error(ER_SYSTEM_KEY_ROTATION_MAX_KEY_ID_EXCEEDED, MYF(0));
-                MYSQL_YYABORT;
-              }
-              $$= NEW_PTN PT_alter_instance(ROTATE_INNODB_SYSTEM_KEY, EMPTY_CSTR, $5);
-            }
-            else
-            {
-              YYTHD->syntax_error_at(@2);
-              MYSQL_YYABORT;
-            }
-          }
-          | ROTATE_SYM ident_or_text SYSTEM_SYM KEY_SYM
-          {
-            if (is_identifier($2, "REDO"))
-            {
-              $$= NEW_PTN PT_alter_instance(ROTATE_REDO_SYSTEM_KEY, EMPTY_CSTR, 0);
-=======
-              $$= NEW_PTN PT_alter_instance(ROTATE_BINLOG_MASTER_KEY, EMPTY_CSTR);
->>>>>>> Percona-Server-8.0.31-23
```

Comment:
Please remove support for ROTATE_INNODB_SYSTEM_KEY and ROTATE_REDO_SYSTEM_KEY in parser.

Reply:
Done.